src/elements/tabs/tab-group/tab-group.component.ts

```diff
@@ -373,9 +188,9 @@ class SbbTabGroupElement extends SbbHydrationMixin(LitElement) {
     }
 
     if (isArrowKeyPressed(evt)) {
-      const current: number = enabledTabs.findIndex((t: InterfaceSbbTabGroupTab) => t.active);
+      const current: number = enabledTabs.findIndex((t: SbbTabLabelElement) => t.active);
       const nextIndex: number = getNextElementIndex(evt, current, enabledTabs.length);
-      enabledTabs[nextIndex]?.tabGroupActions?.select();
+      enabledTabs[nextIndex]?.select();
       enabledTabs[nextIndex]?.focus();
       evt.preventDefault();
```

Comment:
Is preventing the keyboard event really needed? I'ts not optimal as a consumer can also be interested in a keyevent.